test/http.spec.js

```diff
@@ -0,0 +1,29 @@
+import nock from 'nock';
+import path from 'path';
+import downloadFile from '../src/download-file';
+import fs from 'fs';
+import expect from 'expect';
+
+nock('http://local.foo.com')
+  .persist()
+  .get('/image-1.jpg')
+  .replyWithFile(200, path.join(__dirname, 'fixtures/images/image-1.jpg'), {
+    'content-type': 'image/jpg'
+  });
+
+describe('HTTP Server', () => {
+  it('should create an array of objects when passing an array of relative paths', async () => {
+    const file = await downloadFile({
+      host: 'http://local.foo.com',
+      path: 'image-1.jpg',
+      fileName: 'test/whatever-1.jpg'
+    });
+
+    expect(file).toEqual({
+      fileName: 'test/whatever-1.jpg',
```

Comment:
The fixtures folder can have the base images that are gonna be used and we can just reuse them with different names creating endpoints in the nock server. Is that what you mean?